test/qpy_compat/test_qpy.py

```diff
@@ -833,13 +833,19 @@ def generate_v12_expr():
     return [index, shift]
 
 
-def generate_circuits(version_parts, current_version, load_context=False):
-    """Generate reference circuits.
+def generate_v14_expr():
+    """Circuits that contain expressions and types new in QPY v14."""
+    from qiskit.circuit.classical import expr, types
+
+    float_expr = QuantumCircuit(name="float_expr")
+    with float_expr.if_test(expr.less(1.0, 2.0)):
+        pass
+
+    return [float_expr]
 
-    If load_context is True, avoid generating Pulse-based reference
-    circuits. For those circuits, load_qpy only checks that the cached
-    circuits can be loaded without erroring."""
 
+def generate_circuits(version_parts):
+    """Generate reference circuits."""
```

Comment:
You've got a borked merge here - that's what's breaking the QPY tests.

Reply:
whoops!

Reply:
Resolved in 0f14cca4e604c025fe1b1c09c6d85743c6b45014